ViewModel/MainViewModel.cs

```diff
@@ -122,8 +137,7 @@ private async Task GenerateDiff()
             catch(Exception ex)
             {
                 _logService.Error("Generate Diff error. ", ex);
-                //show error to user.
-                throw;//TODO: REMOVE THROW WHEN USR ERROR IS COMPELETED
+                MessageBox.Show("Klaida generuojant nesutapimų ataskaitą","Klaida",MessageBoxButton.OK);                
```

Comment:
Provide exception message text in new line, e.g. in quotes. User might have some insight what happened. maybe user fixable